### ISSUE TITLE
Missing `List` in the generated code

### ISSUE THREAD

**Issue by huan** [danielgtaylor/python-betterproto#218]
betterproto version: v2.0.0b2 

Reproduce steps:

```sh
git clone git@github.com:wechaty/grpc.git
cd grpc/python
make install
make generate
cat src/wechaty_grpc/wechaty/__init__.py
```

We generated the following code:

```py
# ...
from typing import AsyncIterable, AsyncIterator, Iterable, Optional, Union
# ...
    async def contact_phone(
        self, *, contact_id: str = "", phone_list: Optional[List[str]] = None
    ) -> "puppet.ContactPhoneResponse":
# ...
```

As we can see: there's a `List` in the code, but there's no `List` imported from `typing`.

Link to: https://github.com/wechaty/grpc/issues/120

**Comment by Gobot1234** [comment]
This is a known issue and should be fixed by #183

**Comment by huan** [comment]
Got it, thank your very much!
